syncto/tests/test_functional.py

```diff
@@ -0,0 +1,153 @@
+import mock
+from uuid import uuid4
+
+from cliquet.errors import ERRORS
+from cliquet.tests.support import FormattedErrorMixin
+
+from syncto import AUTHORIZATION_HEADER, CLIENT_STATE_HEADER
+
+from .support import BaseWebTest, unittest
+
+
+COLLECTION_URL = "/buckets/syncto/collections/{collection_id}/records"
+RECORD_URL = "/buckets/syncto/collections/{collection_id}/records/{record_id}"
+
+
+class FunctionalTest(FormattedErrorMixin, BaseWebTest, unittest.TestCase):
+
+    def __init__(self, *args, **kwargs):
+        super(FunctionalTest, self).__init__(*args, **kwargs)
+        self.collection_url = COLLECTION_URL.format(collection_id="tabs")
+        self.record_url = RECORD_URL.format(collection_id="tabs",
+                                            record_id=uuid4())
+
+    def test_authorization_header_is_required_for_collection(self):
+        resp = self.app.get(
+            self.collection_url.format(collection_id="tabs"),
```

Comment:
nit: maybe you can store collection_url and record_url once, since it is always used with ``'tabs'`` in the test suite